Source/DataSources/EllipseGeometryUpdater.js

```diff
@@ -196,8 +196,10 @@ define([
     EllipseGeometryUpdater.prototype._getIsClosed = function(options) {
         var height = options.height;
         var extrudedHeight = options.extrudedHeight;
-        var isExtruded = defined(extrudedHeight);
-        return (!isExtruded && height === 0) || (isExtruded && extrudedHeight !== height);
+        if (defined(extrudedHeight)) {
+            return extrudedHeight !== height || height === 0;
+        }
+        return height === 0;
```

Comment:
Same comment, maybe add a `GeometryUpdater._extrudedIsClosed` helper function that derived functions can all?

Reply:
Ah, it's different for Polygon and Rectangle, so a helper function would be overkill.